### PR TITLE
Displays build version of input ISO

### PR DESCRIPTION
**Pull request by Warrentheo**
Type of Change
 Minor update (Console output addition)
Description
This PR adds a Write-Host statement to display the Windows Image version in the console. The change outputs the retrieved image version ($imgVersion) to provide visibility into the version details directly within the script execution output.

Example output:
The Windows Image Build Version is: [version number]

This modification is particularly useful for users who need to confirm the version of the Windows Image being processed.

Testing
Testing involved running the script to ensure that the Write-Host statement correctly displays the image version without causing any errors or affecting existing functionality.

Impact
This change has no impact on performance, dependencies, or behavior beyond adding a simple console output. It provides helpful feedback for users reviewing the image details in the script.

Issue related to PR
N/A

Additional Information
N/A

Checklist
 My code adheres to the coding and style guidelines of the project.
 I have performed a self-review of my own code.
 I have commented my code, particularly in hard-to-understand areas.
 I have made corresponding changes to the documentation.
 My changes generate no errors/warnings/merge conflicts.

### CODE REVIEW

File: functions/microwin/Invoke-Microwin.ps1 [hunk 1]
```diff
@@ -76,6 +76,7 @@ public class PowerManagement {
     }
 
     $imgVersion = (Get-WindowsImage -ImagePath $mountDir\sources\install.wim -Index $index).Version
+    Write-Host "The Windows Image Build Version is: $imgVersion"
```
Comment (CodingWonders) [inline functions/microwin/Invoke-Microwin.ps1:79]:
```suggestion
    Write-Host "Version of selected image: $imgVersion"
```

While this works correctly, I would rephrase this a bit

**Review by CodingWonders — CHANGES_REQUESTED** [review]

File: functions/microwin/Invoke-Microwin.ps1 (hunk 1 above)
Comment (CodingWonders) [reply functions/microwin/Invoke-Microwin.ps1:79]: OK. It is a good change, regardless

Comment (Warrentheo) [reply functions/microwin/Invoke-Microwin.ps1:79]: I do agree it could probably do with a rewording from my phrasing, best one would be a something that translates build numbers into actual version numbers, build such and such is Win11 23H2 for instance, but then you have to maintain that for future changes, so not ideal...

**Review by ChrisTitusTech — APPROVED** [review]
